### ISSUE TITLE
Getting subtitle list at HLS

### ISSUE THREAD

**Issue by ahmetberkinc** [google/ExoPlayer#7744]
Hello, I'm playing HLS type video using exoplayer. I can access audio data via player's trackgroups and make assignments. Is it possible to see what subtitles are available in the video and make assignments by using setPreferredTextLanguage()?

**Comment by andrewlewis** [comment]
I think the short answer is yes. The text tracks should be visible in the track groups, and you can use constraint-based track selection APIs (track selector parameters) to specify a policy like `setPreferredTextLanguage`. If you try doing what you describe do you run into any issues?

**Comment by ahmetberkinc** [comment]
I wanted to access audio and subtitle files in HLS data via streamurl and change audio and subtitle files using PopupMenu.
I did not have any problems with the audio files, but I had difficulty because I could not find enough resources for the subtitle process.
Then I made the necessary assignments by accessing and parsing trackgroups using trackSelector.getCurrentMappedTrackInfo().GetTrackGroups (2); (2 for text / subtitle). This is how I solved the problem and run the application.
Thank you so much!
![Screenshot_16](https://user-images.githubusercontent.com/40149789/90134402-ca1e8300-dd79-11ea-8ca1-f4887e6c9b3c.png)
Screenshot of the application.
